duckduckgo/iOS
# Update animated fire prototype

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/275938474052467

**Description**:
Replace previous static prototype animation with latest moving prototype.
NOTE: This this is NOT the final animation. We are waiting on final assets from the animator and need to implement a proper UIViewController transition.

**Steps to test this PR**:
1. Press the fire button
2. Ensure latest animation is displayed and fills the width of the screen irrespective of device


###### Reviewer Checklist:


###### PR DRI Checklist:
